content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,124 @@
+
+    - Run your project's Asset Processor to begin processing the assets.
+
+<!-- [Question] Is it accurate to say that both methods can complete this task? Is one method recommended? If so, why? -->
+
+{{< caution >}} Ensure that the Asset Processor or Asset Processor Batch processes all of your assets without failure. {{< /caution >}}
+
+
+## Create a project game release layout
+
+You can create a project game release layout by using a non-monolithic or monolithic build. A monolithic release layout only contains files that're necessary to run the project, such as the Game Launcher, packaged assets, and dependent `.dll` files. A non-monolithic release layout contains the same files needed to run the project, as well as the objects that you use for development such as tools, shader compilers, and all `.dll` files.
+
+In this step, you will use CMake to build either a monolithic or a non-monolithic project and specify the `INSTALL` target and `release` configuration. This handles several tasks:
+
+- Creates an `install` directory that will contain your release layout.
+
+- Packages an `engine.pak` file out of the product assets in your `Cache/pc` directory. 
+
+- Builds the Game Launcher and other supporting files.
+
+
+### Monolithic project
+
+1. Set up a new CMake build directory, `windows_mono`, for your monolithic build in your project directory. We recommend keeping this build directory separate from your non-monolithic build directory. Specify a monolithic build by enabling the `-D` option, `LY_MONOLITHIC_GAME`.
+
+    ```cmd
+    cmake -B build/windows_mono -S. -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_MONOLITHIC_GAME=1
+    ```
+
+2. Run the CMake build configuration in your monolithic build directory. This generates a Game Launcher and packages your project's assets in the `Cache` directory into `engine.pak`.
+
+    ```cmd
+    cmake --build build/windows_mono --target INSTALL --config release
+    ```
+
+The result is a project release install directory that's located at `install/bin/windows/release/monolithic`.
+
+
+### Non-monolithic projects
+
+1. Run the CMake build configuration in your non-monolithic build directory, `build/windows_vs2019`.
+
+    ```cmd
+    cmake --build build/windows_vs2019 --target INSTALL --config release
+    ```
+
+The result is a project release install directory that's located at `install/bin/windows/release/Default`.
+
+
+## Run the game launcher
+
+Now that you've packaged your assets and built your project game release layout, you are ready to run your project's Game Launcher.
+
+1. Open a command line window and change to your project's release build install directory.
+
+2. Run the Game Launcher. Supply the path to the level that the project starts with. This opens the Game Launcher and loads the starting level.
+    
+    - **For a monolithic project**:
+ 
+        ```cmd
+        ./MyProject.GameLauncher.exe --project-path="MyProject\install\bin\Windows\release\Monolithic"
```

Comment:
`--project-path` is no longer needed

Reply:
+1 Because the --project-path is no longer needed, we don't have to show two different commands, we can just show the one command to run the game launcher .exe